README.rst

```diff
@@ -111,7 +108,7 @@ Add the middleware to the :code:`MIDDLEWARE` setting (To ensure the GUID to be i
      ]
 
 
-Add a filter to your ``LOGGING``:
+- Add a filter to your ``LOGGING``:
```

Comment:
Not sure it's possible to indent without some sort of bullet, but trying a different alternative. Agree the bulleted list looks bad.

Reply:
You broke the last code block example in one of the later commits. I’ll build the read the docs locally tomorrow and have a look there before approving too. 

Still not sure about the bullet list. Maybe numbered list would be better, since it’s a step by step. If it still looks a bit off I’d say just remove the indent and keep the new text.

Reply:
![bilde](https://user-images.githubusercontent.com/5310116/74012252-1384b480-498a-11ea-8492-f31699f2cfcc.png)

rtfd build also fails. See [rtfd.io](https://django-guid.rtfd.io/en/latest/publish.html#read-the-docs) for information on how to build it locally.

Reply:
I've my edits :) Feel free to jump in and make any changes you like!

Reply:
Resolved the issue above.